shared/chat/conversation/input.desktop.js

```diff
@@ -52,12 +54,11 @@ class Conversation extends Component<void, Props, State> {
       return
     }
 
-    // Allow copy
-    if (ev.metaKey && ['Meta', 'c'].includes(ev.key)) {
-      return
+    const isPasteKey = ev.key === 'v' && (ev.ctrlKey || ev.metaKey)
+    const isValidSpecialKey = ['Backspace', 'Delete', 'ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown', 'Enter'].includes(ev.key)
+    if (ev.type === 'keypress' || isPasteKey || isValidSpecialKey) {
+      this._input.focus()
```

Comment:
Does all this happen in the right order to have a `Backspace` successfully delete a character in the `_input` buffer, or will the user have to press `Backspace` again after the focus change?